actions/merger/entrypoint.sh

```diff
@@ -13,11 +13,12 @@ SWITCH_DIRECTORIES=${SWITCH_DIRECTORIES:-false}
 
 if ${SWITCH_DIRECTORIES} ; then
   cd "${GITHUB_WORKSPACE}"/"${GITHUB_REPO}"
+  git config --global --add safe.directory "${GITHUB_WORKSPACE}"/"${GITHUB_REPO}"
```

Comment:
Just curious, where are these environment variables set?  (Definitely prefer using them throughout!)

Reply:
- GITHUB_WORKSPACE is set automatically by github actions. (List of default variables [here](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables)). Note: In this documentation, it shows that GITHUB_WORKSPACE actually maps to something like '/home/runner/work/my-repo-name/my-repo-name', which is different than what we were originally setting the safe directory path to (/github/workspace). The [github documentation](https://github.com/actions/checkout/issues/766) actually suggests using the env variable instead and from the test runs, it appears to still work. 
- GITHUB_REPO is an environment variable that [we set up for this particular action](https://github.com/broadinstitute/datarepo-actions/tree/master/actions/merger#environment-variables). It has to be set when calling this step.

Reply:
Ah awesome.  I like the fact that this ends. up not needing to be an input to the action